Tiltfile

```diff
@@ -5,20 +5,19 @@ read_file('api/api.py')
 local('make api')
 
 k8s_yaml('serve.yaml')
-repo = local_git_repo('.')
 
 img = fast_build('gcr.io/windmill-public-containers/tilt-site',
                  'base.dockerfile',
                  'bundle exec jekyll serve --config _config.yml,_config-dev.yml')
-img.add(repo.path('src'), '/src/')
+img.add('./src', '/src/')
```

Comment:
Why do we need the `./`?

Reply:
i mean, we don't -- imo it makes the relative-ness of the path more apparent, and also makes it more apparent that this arg is a path and not some other random thing (this is maybe not SO relevant in this example, but for instance i think in a call to
```
docker_build('gcr.io/stuff/thing', 'thing')
```
it's kind of ambiguous what that second arg is (is it maybe the name we assign to the tilt resource for this image? is it...??!) but a call to
```
docker_build('gcr.io/stuff/thing', './thing')
```
makes it v obvious that the second arg is a path, and then it's easier to reason about what it means

Reply:
I think @dbentley expressed a preference for leaving `./` out of paths, but for the record I also like being explicti here.